ArduCopter/Parameters.cpp

```diff
@@ -454,6 +454,13 @@ const AP_Param::Info Copter::var_info[] = {
     // @Increment: 0.1
     // @User: Advanced
     GSCALAR(acro_balance_pitch,     "ACRO_BAL_PITCH",   ACRO_BALANCE_PITCH),
+
+    // @Param: SURF_TRACK_MODE
+    // @DisplayName: Surface Tracking Mode
+    // @Description: which surface to track in surface tracking
+    // @Values: 0: Do not track, 1: Ground, 2: Ceiling
+    // @User: Advanced
+    GSCALAR(surf_track_mode,     "SURF_TRACK_MODE",   SURF_TRACK_MODE_DEFAULT),
```

Comment:
I would not bother with the default define, doubt anyone will ever use it. 
```suggestion
    GSCALAR(surf_track_mode,     "SURF_TRACK_MODE",   Surface::GROUND),
```

Reply:
I too was not sure about it. I have removed it now. Please let me know in case of more discrepancies.